simplifieddata/extend_right.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
This directory name won't age well. Since this file is used in unit testing, perhaps put it right next to the unit test source file and load it as a resource? But then perhaps we should also create a new subproject for simple-arc-world? @dwaxemberg will know what to do!

Reply:
Created a new module for the simple case